app/static/src/app/directives/tooltip.ts

```diff
@@ -1,13 +1,50 @@
-import { Tooltip } from "bootstrap";
 import { DirectiveBinding } from "vue";
+import { Tooltip } from "bootstrap";
+
+interface ToolTipContent {
+    content?: string,
+    trigger?: "hover" | "focus" | "click" | "click focus" | "click hover" | "click hover focus" | "hover focus",
```

Comment:
I guess we don't really need all these options yet, but it's good to have them available, particularly if we end up spinning this out into its own package.

Reply:
yh i thought we may as well approach all the settings of the bootstrap tooltip

Reply:
Is it worth pulling these from the bootstrap types? https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/bootstrap/v4/index.d.ts - if the aim is to match what is there it will make it clearer, but also guarantees that you match things properly with no typos

Reply:
i think this is a good idea! i have installed that package